scheduler/src/main/resources/application.properties

```diff
@@ -15,8 +15,12 @@ autoscaler.scalingengine.url=http://test-scalingengine-test
 
 #ssl
 server.ssl.key-store=../test-certs/scheduler.p12
-caCert=../test-certs/autoscaler-ca.crt
 server.ssl.key-alias=scheduler
 server.ssl.key-store-password=123456
-#server.ssl.key-password=123456
 server.ssl.key-store-type=PKCS12
+client.ssl.keyStore=../test-certs/scheduler.p12
+client.ssl.keyStorePassword=123456
+client.ssl.trustStore=../test-certs/scheduler.truststore
```

Comment:
There will be one truststore for scheduler. Now for keystore there can be two, one for server and one for  client. Having one keystore is also fine. But still better to have separate properties for server and client as the server .ssl... properties are are required for server authentication and are recognised by springboot. Also when doing client authentication when talking to scaling engine specifying server.ssl.. properties are a misnomer. What are your thoughts?